taxi/views.py

```diff
@@ -1,3 +1,15 @@
 from django.shortcuts import render
+from .models import Driver, Manufacturer, Car
 
-# Create your views here.
+def index(request):
+    num_drivers = Driver.objects.count()          # Підрахунок водіїв
+    num_manufacturers = Manufacturer.objects.count()  # Підрахунок виробників
+    num_cars = Car.objects.count()                # Підрахунок автомобілів
+
+    context = {
+        'num_drivers': num_drivers,
+        'num_manufacturers': num_manufacturers,
+        'num_cars': num_cars,
+    }
+
+    return render(request, 'taxi/index.html', context) #templates???
```

Comment:
The comment questioning the template path is unnecessary. The path 'taxi/index.html' is correct according to the project structure.